_config.yml

```diff
@@ -39,7 +39,7 @@ collections_dir: docs
 
 collections:
   tokens:
-    sort_by: order
+    sort_by: title
```

Comment:
I don't think we want these sorted by `title` it make the different sizing of the fonts into a strange order

Reply:
How do you want the whole page ordered?

Reply:
The page should be ordered by the order the sass rules are defined:

  "ultra-light": 200,
  "light": 300,
  "normal": 400,
  "bold": 700,
  "heavy": 900
  
    "xxs": 0.625rem,
  "xs": 0.75rem,
  "sm": 0.875rem,
  "md": 1rem,
  "lg": 1.5rem,
  "xl": 2rem,
  "xxl": 3rem,